.changeset/thin-bags-repair.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/admin": minor
+---
+
+Add the `useDataGridExcelExport` hook for exporting data from a `DataGrid` to an excel file 
+
+The hook returns an `exportApi` encompassing: 
+
+-   `exportGrid`: a function to generate and export the excel file
+-   `loading`: a boolean indicating if the export is in progress
+-   `error`: an error message when the export has failed
```

Comment:
Should this wording be changed? Because of https://github.com/vivid-planet/comet/pull/1948/commits/0375257788278765d36a874d122b30dfbca4a517

```suggestion
-   `error`: an error when the export has failed
```